jordan_bundy/lib/routes.js

```diff
@@ -0,0 +1,49 @@
+var fs = require('fs');
+
+var routes = {};
+var routeList = '';
+
+routes['/'] = {
+  GET: {
+    status: 200,
+    contentType: 'text/html',
+    data: fs.readFileSync(__dirname + '/../public/index.html').toString()
+  }
+};
+
+routes['/name'] = {
+  GET: {
+    status: 200,
+    contentType: 'text/plain',
+    data: 'Hello there Mr. '
+  },
+  POST: {
+    
+  }
+};
+
+routes['/time'] = {
+  GET: {
+    status: 200,
+    contentType: 'text/plain',
+    data: new Date().toString()
+  }
+};
+
+Object.keys(routes).forEach(function(route) {
```

Comment:
unfortunately i spent more time on building my router module than completing the requirements of the assignment!!